integration/projects/test_models_are_executed/test_models_are_executed.py

```diff
@@ -8,7 +8,7 @@ def test_success(dry_run_result: IntegrationTestResult):
 
 def test_ran_correct_number_of_nodes(dry_run_result: IntegrationTestResult):
     report = assert_report_success(dry_run_result)
-    assert report.node_count == 3
+    assert report.node_count == 4
```

Comment:
On top of this can you add another test case similar to `test_case_timestamp_column_succeeds` in `test_project_with_snapshots.py`. Something just to assert that `node_with_all_column_types` has the expected column names of `my_string, my_bytes, my_integer` etc.

Reply:
I've done that now.